### PR TITLE
query for all servers matching the specified name

### PR DESCRIPTION
**Pull request by serge17**

### CODE REVIEW

File: src/main/java/com/contrastsecurity/maven/plugin/VerifyContrastMavenPluginMojo.java [hunk 1]
```diff
@@ -32,12 +26,19 @@ public void execute() throws MojoExecutionException {
...
+        if (StringUtils.isNotBlank(serverName)) {
+            serverIds = getServerId(contrast, applicationId);
+        }
 
         TraceFilterForm form = new TraceFilterForm();
         form.setSeverities(getSeverityList(minSeverity));
         form.setAppVersionTags(Collections.singletonList(computedAppVersion));
-        form.setServerIds(Arrays.asList(serverId));
+
+        if (serverIds != null) {
```
Comment (gmoore) [inline src/main/java/com/contrastsecurity/maven/plugin/VerifyContrastMavenPluginMojo.java]: @serge17 Looking good. Can you stand up a test for this to make sure we're filling out the filterForm correctly? I don't think we have tests around the Verify yet.

**Review by gmoore — CHANGES_REQUESTED** [review]

File: src/main/java/com/contrastsecurity/maven/plugin/VerifyContrastMavenPluginMojo.java (hunk 1 above)
Comment (serge17) [reply src/main/java/com/contrastsecurity/maven/plugin/VerifyContrastMavenPluginMojo.java]: @gmoore A test class added.

**Comment by gmoore** [comment]
@serge17  Tests looking good. Can you add tests for `setSeverities` and `setAppVersionTags` ?

**Comment by serge17** [comment]
@gmoore Added 2 tests.

**Review by gmoore — APPROVED** [review]

**Pull request merged**